opencti-platform/opencti-front/lang/front/de.json

```
@@ -3077,6 +3077,7 @@
   "Series attribute": "Serienattribut",
   "Server capacity": "Serverkapazität",
   "Server ingestion is paused until the size of messages is reduced under max capacity": "Server-Ingestion wird angehalten, bis die Größe der Nachrichten unter die maximale Kapazität gesenkt wird",
+  "Service accounts will not have any password & a randomized email adress will be provided, if not provided.Service account will not be able to receive notifications.Service account will also automatically pertains to the main platform org.": "Service-Konten haben kein Passwort & eine zufällige E-Mail-Adresse wird bereitgestellt, wenn nicht angegeben.Service-Konto wird nicht in der Lage sein, Benachrichtigungen zu erhalten.Service-Konto wird auch automatisch auf die Hauptplattform org gehört.",
```

```suggestion
  "Service accounts do not have any password and a randomized email address will be generated, if not provided on creation. Service account do not receive notifications. Service account pertains automatically  to the main platform organization.": "Service-Konten haben kein Passwort & eine zufällige E-Mail-Adresse wird bereitgestellt, wenn nicht angegeben.Service-Konto wird nicht in der Lage sein, Benachrichtigungen zu erhalten.Service-Konto wird auch automatisch auf die Hauptplattform org gehört.",
```
My personal advice for end users texts:
- no abbreviation
- use present time as much as possible

---

Note that for this kind of text, it's also possible to use a short text key (like : `"Service accounts do not have any password and a randomized email address will be generated, if not provided on creation..."`) and then modification must be done on the `en`  file.